pages/login.vue

```diff
@@ -75,19 +75,22 @@ export default Vue.extend({
   async beforeMount() {
     try {
       this.publicUsers = (await this.$api.user.getPublicUsers({})).data;
-      const brandingData = (await this.$api.branding.getBrandingOptions()).data;
-      this.disclaimer = brandingData.LoginDisclaimer || '';
     } catch (error) {
-      console.error('Unable to get public users:', error);
+      this.pushSnackbarMessage({
+        message: this.$t('unableGetPublicUsers'),
+        color: 'error'
+      });
     }
+    this.getLoginDisclaimer();
```

Comment:
We are still going to have to wait for `getPublicusers` before we can get the login disclaimer

Reply:
@Camc314 Hmm then I don't understand what you meant initially. Can you detail a little more please?

Reply:
Sure, No worries. Basically, in there would be two methods. `getLoginDisclaimer` and `getPublicUsers`. On `beforeMount` or `created` (whichever we are already using), both of the methods would be called.

Now, since we wouldn't have to fetch, and wait for the public users before getting the login the disclaimer, we could fetch both simultaneously, and process the responses in their relevant functions.

Hopefully that makes more sense.